packages/web/src/hooks/useUserProfilePicture.ts

```diff
@@ -28,6 +32,27 @@ export const useUserProfilePicture = (
   })
 }
 
+export const useUserProfilePicture2 = (
```

Comment:
maybe a more descriptive name or a comment suggesting the different args

Reply:
> maybe a more descriptive name or a comment suggesting the different args

Yeah agreed, may also deprecate the old one